src/Microsoft.TemplateEngine.Cli/Installer.cs

```diff
@@ -189,6 +200,23 @@ private void InstallRemotePackages(List<Package> packages)
             InstallLocalPackages(newLocalPackages);
         }
 
+        private void InstallGitSources(List<GitSource> gitSources)
+        {
+            _paths.CreateDirectory(_paths.User.ScratchDir);
+
+            List<string> newLocalPackages = new List<string>();
+
+            foreach (GitSource source in gitSources)
+            {
+                string targetPath = $"{_paths.User.ScratchDir}/{source.RepositoryName}";
+                ExecuteProcess("git", "clone", source.GitUrl, targetPath);
```

Comment:
Yeah, I had the same thought but figured most folks will have git so we are incrementally adding value here.  I like the Installer idea but its a larger design/development effort.  Do you think this is a must have at this point?

I will add a check for the exit code.